packages/google_maps_flutter/lib/src/google_maps_controller.dart

```diff
@@ -459,11 +476,101 @@ class GoogleMapController {
 
   /// Applies a `cameraUpdate` to the current viewport.
   Future<void> moveCamera(CameraUpdate cameraUpdate) async {
-    // TODO : implement for google_map_tizen if necessary
-    // assert(_googleMap != null, 'Cannot update the camera of a null map.');
-    // return _applyCameraUpdate(_googleMap!, cameraUpdate);
-
     assert(_widget != null, 'Cannot update the camera of a null map.');
+    _applyCameraUpdate(cameraUpdate);
+  }
+
+  // Translates a [CameraUpdate] into operations on a [Javascript].
+  Future<void> _applyCameraUpdate(CameraUpdate update) async {
+    final List<dynamic> json = update.toJson() as List<dynamic>;
```

Comment:
Don't you need a type check or length like the one below?

```dart 
assert(latlng['lat'] is num && latlng['lng'] is num);
```

Reply:
Do you mean json[0]'s type?
I am not sure but if it is other type or other value, I think it may jump to default statement.
Should I add an assert statement?

Reply:
Whatever it is, I don't care.  I've seen you usually put assertions in the parts you're currently implementing.
So I just wanted these to be consistent with other implementations.

Reply:
I think it will be able to handled by default statement. Hence, I think it is not necessary here.